aioesphomeapi/model.py

```diff
@@ -820,10 +820,42 @@ class MediaPlayerCommand(APIIntEnum):
     UNMUTE = 4
 
 
+class MediaPlayerFormatPurpose(APIIntEnum):
+    DEFAULT = 0
+    ANNOUNCEMENT = 1
+
+
+@_frozen_dataclass_decorator
+class MediaPlayerSupportedFormat(APIModelBase):
+    format: str
+    sample_rate: int
+    num_channels: int
+    purpose: MediaPlayerFormatPurpose | None = converter_field(
+        default=MediaPlayerFormatPurpose.DEFAULT,
+        converter=MediaPlayerFormatPurpose.convert,
+    )
+
+    @classmethod
+    def convert_list(cls, value: list[Any]) -> list[MediaPlayerSupportedFormat]:
+        ret = []
+        for x in value:
+            if isinstance(x, dict):
+                if "type_" in x and "type" not in x:
+                    x = {**x, "type": x["type_"]}
```

Comment:
I don't think this is needed. It's only used on another message where there actually is a `type` field
```suggestion
```